resources/views/admin.blade.php

```diff
@@ -108,18 +134,47 @@ function deleteEmployee(id){
                 }
 
                 //Create Employee Axios
-                $('#createForm').submit(function (e) {
+                $('#modal-form').submit(function (e) {
 
                     e.preventDefault();
-                    let $data = $('#createForm').serialize()
-                    let $dataArray = $('#createForm').serializeArray()
+                    let $data = $('#modal-form').serialize()
                     axios.post('employee', $data )
                     .then(function (response){
-                        $('#createForm').trigger("reset");
+                        $('#modal-form').trigger("reset");
                         $('#btn-modal-close').click();
                         table.draw();
                     });
                 });
+
+                let $id = null
+
+                //Edit Employee
+                $('#myTable tbody').on('click', 'button[id^="btn-edit"]', function (e) {
+                    const $data =  table.row( $(this).parents('tr') ).data();
+                    $id  = $data.id
+                    
+                    $('#modal-edit').each(function (){
+                        const $inputs =  $(this).find(':input');
+                        $($inputs[1]).val($data.name);
```

Comment:
better to use class name or element id when you set values

Reply:
changes made please review again @Raza9798 #22